extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -289,24 +289,24 @@ public function executeBackgroundImageUpload() {
 	 * @param UploadBase $upload
 	 * @return array
 	 */
-	private function uploadImage($upload) {
+	private function uploadImage( $upload ) {
 		global $wgRequest, $wgUser, $wgEnableUploads;
 
-		$uploadStatus = array("status" => "error");
+		$uploadStatus = [ "status" => "error" ];
 
-		if ( empty( $wgEnableUploads )) {
+		if ( empty( $wgEnableUploads ) ) {
 			$uploadStatus["errors"] = [ wfMessage( 'themedesigner-upload-disabled' )->plain() ];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change the output format of this i18n string away from plain(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)